src/officehours_api/management/commands/phase_out_backends.py

```diff
@@ -0,0 +1,49 @@
+from typing import Set
+
+from django.conf import settings
+from django.core.management.base import BaseCommand
+
+from officehours_api.backends import __all__ as IMPLEMENTED_BACKEND_NAMES
```

Comment:
I'd be curious to know if this seems like a good idea or not. I realized that defining the list of implemented backends in `settings.py` actually didn't make that much sense, and this is probably a more straightforward approach that promotes keeping things in sync.

Reply:
Yeah, this makes the most sense.